src/main/resources/messages/RsBundle.properties

```diff
@@ -253,3 +253,5 @@ inspection.RedundantSemicolons.fix.name=Remove unnecessary trailing semicolons
 
 inspection.DoubleMustUse.description=This function has a `#[must_use]` attribute, but returns a type already marked as `#[must_use]`
 inspection.DoubleMustUse.FixRemoveMustUseAttr.name=Remove `#[must_use]` from the function
+
+inspection.PathStatementsInspection.description.no.effect=Path statement with no effect
```

Comment:
Considering #9555 we may want to change this message for now to "Path statement drops value or has no effect" but since it's kinda confusing I'm not sure it's worth it.